.env

```diff
@@ -0,0 +1 @@
+API_KEY=a99996dd4383ac814120045852e03bd6
```

Comment:
unfortunately, this key has already been pushed to GitHub and is thus exposed. 

I strongly suggest, you create / request a new API Key and update your `.env` file accordingly